core-service/src/test/java/com/crmfoodestablishment/coreservice/service/MenuServiceTest.java

```diff
@@ -0,0 +1,125 @@
+package com.crmfoodestablishment.coreservice.service;
+
+import com.crmfoodestablishment.coreservice.entity.Menu;
+import com.crmfoodestablishment.coreservice.entity.Season;
+import com.crmfoodestablishment.coreservice.exception.MenuNotFoundException;
+import com.crmfoodestablishment.coreservice.repository.MenuRepository;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import java.util.Optional;
+import static org.assertj.core.api.Java6Assertions.*;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.*;
+
+@ExtendWith(MockitoExtension.class)
+class MenuServiceTest {
+
+    @Mock
+    MenuRepository menuRepository;
+
+    @InjectMocks
+    private MenuService service;
+
+
+    private static Menu createMenu() {
+        Menu menu = new Menu(
+                "summer menu",
+                "new summer menu",
+                Season.SUMMER
+        );
+        return menu;
+    }
+
+
+    @BeforeEach
+    void setUp() {
+        service = new MenuService(menuRepository);
+    }
+
+    @Test
+    void canAddNewMenu() {
```

Comment:
it is better to name test methods like 'shoudAddNewMenu'

Reply:
Got it.